src/Workspaces/Core/Portable/FindSymbols/SymbolTree/SymbolTreeInfo.cs

```diff
@@ -26,7 +26,7 @@ namespace Microsoft.CodeAnalysis.FindSymbols
     /// will still incur a heavy cost (for example, getting the <see cref="IAssemblySymbol"/> root symbol for a
     /// particular project).
     /// </summary>
-    internal partial class SymbolTreeInfo : IChecksummedObject
+    internal partial class SymbolTreeInfo
```

Comment:
there was no point in this being an IChecksummedObject.  That's only used when serializing types to/from oop.  which we don't do with this type.

Reply:
we also weren't doing anything at all interesting with this checksum :)